.github/workflows/flutter_integration_test.yml

```diff
@@ -46,11 +46,39 @@ jobs:
       - name: flutter pub get
         run: flutter pub get
 
+      - name: Gradle cache
+        uses: gradle/gradle-build-action@v2
+
+      - name: AVD cache
+        uses: actions/cache@v3
+        id: avd-cache
+        with:
+          path: |
+            ~/.android/avd/*
+            ~/.android/adb*
+          key: avd-21
+
+      - name: create AVD and generate snapshot for caching
+        if: steps.avd-cache.outputs.cache-hit != 'true'
+        uses: reactivecircus/android-emulator-runner@d94c3fbe4fe6a29e4a5ba47c12fb47677c73656b #pin@v2.28.0
+        with:
+          working-directory: ./flutter/example
+          api-level: 21
+          force-avd-creation: false
+          emulator-options: -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
+          disable-animations: false
```

Comment:
Should this also be `true`? I guess we don't need animation anywhere.
```
          disable-animations: true
```

Reply:
I've noticed that in the actions' repo it is also with enabled animations, but that must be a typo, or does it have a side effect?
https://github.com/ReactiveCircus/android-emulator-runner

Reply:
No idea, I'd keep it as it is tho since it's following the https://github.com/ReactiveCircus/android-emulator-runner example.